README.md

```diff
@@ -168,6 +168,22 @@ jobs:
 - **Description**: A string of extra arguments passed to clang-tidy for use as compiler arguments (like `-std=c++14 -Wall`).
 - Default: ''
 
+#### `tidy-review`
+
```

Comment:
The clang-format reviews are pretty straight forward, the code is either properly formatted or it is not. But clang-tidy reviews are not yet completely finalized. If `lines-changed-only` is not `true`, then some clang-tidy concerns may not be shown to user in the PR reviews.

I could re-work the PR review comments again for this case or we can just flag the `tidy-review` input as
```suggestion

**Beta feature** 🚧

```

Personally, I'm partial to just calling it a beta feature and rolling it out for field testing.